ui/src/shared/components/HistogramChartTooltip.tsx

```diff
@@ -25,18 +27,34 @@ const HistogramChartTooltip: SFC<Props> = props => {
   return (
     <div className="histogram-chart-tooltip" style={tooltipStyle}>
       <div className="histogram-chart-tooltip--column">
-        {data.map(d => (
-          <div key={d.key} style={{color: colorScale(d.group)}}>
-            {d.value}
-          </div>
-        ))}
+        {data.map(d => {
```

Comment:
It took me a sec to spot the diff between this and the next. We could define a another component to put in the map: `HistogramChartTooltipColumnItem` that could take props like the following

```typescript
interface Props {
    groupColor: HistorgramColor
    historgramDatum: HistorgramDatum
    datumName: string  // something like 'value' or 'group'?
}
```

Reply:
since the color is different for each of the different groups, color has to be selected inside each map. i think im going to leave it is for now. but thanks for the suggestion